src/_repobee/command/repos.py

```diff
@@ -401,9 +411,18 @@ def _open_issue_by_urls(
         plug.log.info(msg)
 
 
+class DirectoryLayout(enum.Enum):
+    FLAT = "flat"
+    NESTED_BY_TEAM = "by-team"
+
+    def __str__(self):
+        return str(self.value)
```

Comment:
Dont think i saw this used anywhere but could be good to have anyway

Reply:
i see now that you're actually calling .value directly on line 374 instead using str()

Reply:
This provides the human-readable string representation of the enum, which most notably is used in the help section of the CLI (i.e. when typing `repobee repos clone -h`). Without it, the help section would contain e.g. `DirectoryLayout.FLAT` instead of just `flat` as a choice for the `--dl` option.